pyomo/core/tests/transform/test_strip_bounds.py

```diff
@@ -0,0 +1,107 @@
+"""Tests stripping of variable bounds."""
+import pyutilib.th as unittest
+
+from pyomo.environ import (Binary, ConcreteModel, Integers, NonNegativeReals,
+                           PositiveReals, Reals, TransformationFactory, Var)
+
+
+class TestStripBounds(unittest.TestCase):
+    """Tests stripping of bounds."""
+
+    def test_strip_bounds_maps_exist(self):
+        """Tests if component maps for reversion already exist."""
+        m = ConcreteModel()
+        m.v0 = Var(bounds=(2, 4))
+        m.v1 = Var(domain=NonNegativeReals)
+        m.v2 = Var(domain=PositiveReals)
+        m.v3 = Var(bounds=(-1, 1))
+        m.v4 = Var(domain=Binary)
+        m.v5 = Var(domain=Integers, bounds=(15, 16))
+
+        xfrm = TransformationFactory('core.strip_var_bounds')
+        xfrm.apply_to(m, reversible=True)
+        # At this point, component maps for reversion already exist.
+        with self.assertRaises(RuntimeError):
+            xfrm.apply_to(m, reversible=True)
+
+    def test_strip_bounds(self):
+        """Test bound stripping and restoration."""
```

Comment:
This is nit-picking, but I prefer that test methods NOT use docstrings (use general comments instead).  The problem is that when the test fails (or when nose is running in verbose mode), nose will report the docstring instead of the fully-qualified test name (`module.module.class.method`), making it tough to actually *find* the test in question.

Reply:
This feels like it ought to be more of a nose option setting than a docstring vs. comment issue, because removing the docstrings makes pyflakes angry at me for not having a docstring on a public method.

Reply:
Here is a workaround that allows docstrings on tests, but also reports the test name instead of the docstring. Would this work?

https://www.saltycrane.com/blog/2012/07/how-prevent-nose-unittest-using-docstring-when-verbosity-2/

Reply:
Yes.  I think that might work.  I think we can put into pyutilib.th -- which would fix all of Pyomo.

Reply:
@whart222: do you see any reason not to implement a default `shortDescription` method for `pyutilib.th.pyunit.TestCase`?